src/main/java/zowe/client/sdk/zosjobs/input/SubmitJclParams.java

```diff
@@ -42,11 +43,11 @@ public class SubmitJclParams {
     private Optional<String> internalReaderLrecl;
 
     /**
-     * A string for JCL symbolic substitution
+     * A Map for JCL symbolic substitution
      * <p>
-     * For example, this accepts a string of key/value pairs separated by spaces: SYM=SYM SYM2=SYM2, etc..
+     * For example, this accepts a Map of key/value pairs: {"SYM","SYM"}, {"SYM2","SYM2"}, etc..
```

Comment:
I think we should show an example with value containing a space with double quotes?